<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleToolGUI</class>
 <widget class="QMainWindow" name="ScaleToolGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scale Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>254</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scale Tool</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="parameter_label">
         <property name="text">
          <string>Parameter(s):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="scaling_factor_label">
         <property name="text">
          <string>Scaling Factor:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="parameter_combo">
         <item>
          <property name="text">
           <string>X, Y, Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X, Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX, PY, PZ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX, PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PZ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="scaling_factor">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>1.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="text">
          <string>Apply</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
